---
layout: page
title: Yuchen Yang's publications
---

<div class="navbar">
  <div class="navbar-inner">
      <ul class="nav">
          <li><a href="https://scholar.google.com/citations?user=0CK0320AAAAJ">Google Scholar</a></li>
      </ul>
  </div>
</div>

&dagger; indicates co-first authors; &sect; indicates co-corresponding authors.

### Preprint

Jiang, Y., Li, W., Lindsey-Boltz, L., **Yang, Y.**, Li, Y., Sancar, A. Nucleotide excision repair hotspots of UV-induced DNA damage in the human genome. ***bioRxiv***, 2020. ([link](https://www.biorxiv.org/content/10.1101/2020.04.16.045369v1))

Crowley, C., **Yang, Y.**, Qiu, Y., Hu, B., Lipiński, J., Plewczynski, D., Won, H., Ren, B., Hu, M., Li, Y. FIREcaller: detecting frequently interacting regions from Hi-C data. ***bioRxiv***, 2020. ([link](https://www.biorxiv.org/content/10.1101/619288v1.abstract))

Giusti-Rodríguez, P.<sup>&dagger;</sup>, Lu, L.<sup>&dagger;</sup>, **Yang, Y.<sup>&dagger;</sup>**, Crowley, C., Liu, X., Juric, I., *et al.* Using three-dimensional regulatory chromatin interactions from adult and fetal cortex to interpret genetic results for psychiatric disorders and cognitive traits. ***bioRxiv***, 2020. ([link](https://www.biorxiv.org/content/10.1101/406330v1.full))

### 2020

**Yang, Y.<sup>&dagger;</sup>**, Li, G.<sup>&dagger;</sup>, Qian, H., Wilhelmsen, K., Shen, Y., Li., Y. SMNN: Batch Effect Correction for Single-cell RNA-seq data via Supervised Mutual Nearest Neighbor Detection. ***Briefings in Bioinformatics***, Accepted. ([link](https://www.biorxiv.org/content/10.1101/672261v2.abstract))

**Yang, Y.**, Li, Y., Sancar, A., Oztas, O. (2020) The circadian clock shapes the Arabidopsis transcriptome through regulating alternative splicing and alternative polyadenylation. ***Journal of Biological Chemistry***, 295:7608-7619. ([link](https://www-jbc-org.libproxy.lib.unc.edu/content/295/22/7608))

Feng, X., Xu, S., Li, J., **Yang, Y.**, Chen, Q., Lyu, H., Zhong, C., He, Z., Shi, S. (2020) Molecular adaptation to salinity fluctuation in tropical intertidal environments of a mangrove tree *Sonneratia alba*. ***BMC Plant Biology***, 20:178. ([link](https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-020-02395-3))

Johnson, N., Huang, L., Li, R., Li, Y., **Yang, Y.**, Kim H., et al. (2020) Age-related DNA hydroxymethylation is enriched for gene expression and immune system processes in human peripheral blood. ***Epigenetics***, 15: 294-306. PMID: 31506003. ([link](https://www.tandfonline.com/doi/abs/10.1080/15592294.2019.1666651))

Huh, R.<sup>&dagger;</sup>, **Yang, Y.<sup>&dagger;</sup>**, Jiang, Y., Shen, Y., Li, Y. (2020) SAME-clustering: Single-cell Aggregated Clustering via Mixture Model Ensemble. ***Nucleic Acids Research***, 48: 86-95. PMID: 31777938. ([link](https://academic.oup.com/nar/article/48/1/86/5644992))


### 2019

Li, G., **Yang, Y.**, Van Buren, E., Li, Y. (2019) Dropout Imputation and Batch Effect Correction for Single-Cell RNA-seq Data. ***Journal of Bio-X Research***, 2: 169-177. ([link](https://journals.lww.com/jbioxresearch/Fulltext/2019/12000/Dropout_imputation_and_batch_effect_correction_for.4.aspx))

**Yang, Y.**, Huh, R., Culpepper, H., Lin, Y., Love, M., Li, Y. (2019) SAFE-clustering: Single-cell Aggregated (From Ensemble) Clustering for Single-cell RNA-seq Data. ***Bioinformatics***, 35: 1269-1277. PMID: 30202935. ([link](https://academic.oup.com/bioinformatics/article-abstract/35/8/1269/5092931))

Zhou Q.<sup>&dagger;</sup>, **Yang Y.<sup>&dagger;</sup>**, Yang Z. (2019) Molecular dissection of cadmium-responsive transcriptome profile in a low-cadmium-accumulating cultivar of Brassica parachinensis. ***Ecotoxicology and Environmental Safety***, 176: 85-94. PMID: 30921700. ([link](https://www.sciencedirect.com/science/article/abs/pii/S0147651319303495))

Juric I., Yu M., Abnousi A., Raviram R., Fang R., Zhao Y., Zhang Y., Qiu Y., **Yang Y.**, Li Y., Bing Ren B., Hu M. (2019) MAPS: model-based analysis of long-range chromatin interactions from PLAC-seq and HiChIP experiments. ***PLOS Computational Biology***, 15(4): e1006982. PMID: 30986246. ([link](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006982))

### 2018

**Yang, Y.**, Guo, W., Shen, X., Li, J., Yang, S., Chen, S., He, Z., Zhou R., Shi, S. (2018) Identification and characterization of evolutionarily conserved alternative splicing events in a mangrove genus *Sonneratia*. ***Scientific Reports***, 8: 4425. PMID: 29535339. ([link](https://www.nature.com/articles/s41598-018-22406-6))

Guo, Z., Li, X., He, Z., **Yang, Y.**, Wang, W., Zhong, C., Greenberg, A.J., Wu, C-I, Duke N.C., Shi, S. (2018) Extremely low genetic diversity across mangrove taxa reflects past sea level changes and hints at poor future responses. ***Global Change Biology***, 24: 1741–1748. PMID: 29087007. ([link](https://onlinelibrary.wiley.com/doi/abs/10.1111/gcb.13968))

### 2017

**Yang, Y.**, Li, J., Yang, S., Li, X., Fang, L., Zhong, C., Duke, N., Zhou, R., Shi, S. (2016) Effects of Pleistocene sea-level fluctuations on mangrove population dynamics: A lesson from *Sonneratia alba*. ***BMC Evolutionary Biology***, 17: 22. PMID: 28100168. ([link](https://link.springer.com/article/10.1186/s12862-016-0849-z))

Li, J., **Yang, Y.**, Yang, S., Zhang, Z., Chen, S., Zhong, C., Zhou, R., Shi, S. (2017) Comparative transcriptome analyses of a mangrove tree *Sonneratia caseolaris* and its non-mangrove relatives, *Trapa bispinosa* and *Duabanga grandiflora*. ***Marine Genomics***, 31: 13-15. PMID: 27810366. ([link](https://www.sciencedirect.com/science/article/abs/pii/S1874778716301349))

Zhou, Q., **Yang, Y.**, Shen, C., He, C., Yuan, J., Yang, Z. (2017) Comparative analysis between low- and high-cadmium-accumulating cultivars of *Brassica parachinensis* to identify difference of cadmium-induced microRNA and their targets. ***Plant and Soil***, 420: 223-237. ([link](https://link.springer.com/article/10.1007/s11104-017-3380-0))

Du, Y., Martin, J., McGee, J., **Yang, Y.**, Liu, E., Sun, Y., Geihs, M., Kong, X., Zhou, E., Li, Y., Huang, J. (2017) A SNP panel and online tool for checking genotype concordance through comparing QR codes. ***PLoS ONE***, 12(9): e0182438. PMID: 28926565. ([link](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5604942/))

### 2016

**Yang, Y.**, Duke, N.C., Peng, F., Li, J., Yang, S., Zhong, C., Zhou, R., Shi, S. (2016) Ancient geographical barriers drive differentiation among *Sonneratia caseolaris* populations and recent divergence from *S. lanceolata*. ***Frontiers in Plant Science***, 7: 1618. PMID: 27833634. ([link](https://www.frontiersin.org/articles/10.3389/fpls.2016.01618/full))

Li, J., **Yang, Y.**, Chen, Q., Fang, L., He, Z., Guo, W., Qiao, S., Wang, Z., Guo, M., Zhong, C., Zhou R., Shi. S. (2016) Pronounced genetic differentiation and recent secondary contact in the mangrove tree Lumnitzera racemosa revealed by population genomic analyses. ***Scientific Reports***, 6: 29486. PMID: 27380895. ([link](https://www.nature.com/articles/srep29486))

Fang, L., **Yang, Y.**, Guo, W., Li, J., Zhong, C., Huang, Y., Zhou, R., Shi, S. (2016) De novo assembly of the transcriptome of *Aegiceras corniculatum*, a mangrove species in the Indo-West Pacific region. ***Marine Genomics***, 28: 49–52. PMID: 27296448. ([link](https://www.sciencedirect.com/science/article/abs/pii/S1874778716300526))

Li, X., Duke, N., **Yang, Y.**, Huang, L., Zhu, Y., Zhang, Z., Zhou, R., Zhong, C., Huang, Y., Shi, S. (2016) Re-Evaluation of Phylogenetic Relationships among Species of the Mangrove Genus Avicennia from Indo-West Pacific Based on Multilocus Analyses. ***PLoS ONE***, 11(10): e0164453. PMID: 27716800. ([link](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5055292/))

Wang, S., Chen, Y., **Yang, Y.**, Wu, W., Liu, Y., Fan, Q., Zhou, R. (2016) Phylogenetic relationships and natural hybridization in *Triadica* inferred from nuclear and chloroplast DNA analyses. ***Biochemical Systematics and Ecology***, 64: 142-148. ([link](https://www.sciencedirect.com/science/article/pii/S0305197815002938))

Zhang, Z., He, Z., Xu, S., Li, X., Guo, W., **Yang, Y.**, Zhong, C., Zhou, R., Shi, S. (2016) Transcriptome analyses provide insights into the phylogeny and adaptive evolution of the mangrove fern genus *Acrostichum*. ***Scientific Reports***, 6: 35634. PMID: 27782130. ([link](https://www.nature.com/articles/srep35634))

Shi, L., Li, N., Wang, S., Zhou, Y., Huang, W., **Yang, Y.**, Ma, Y., Zhou, R. (2016) Molecular Evidence for the Hybrid Origin of *Ilex dabieshanensis* (Aquifoliaceae). ***PLoS ONE***, 11(1): e0147825. PMID: 26808531. ([link](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0147825))

### 2015

**Yang Y.**, Yang, S., Li, J., Deng, Y., Zhang, Z., Xu, S., Guo, W., Zhong, C., Zhou, R., Shi, S. (2015) Transcriptome analysis of the Holly mangrove Acanthus ilicifolius and its terrestrial relative, Acanthus leucostachyus, provides insights into adaptation to intertidal zones. ***BMC Genomics***, 16: 605. PMID: 26272068. ([link](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-1813-9))

**Yang, Y.**, Yang, S., Li, J., Li, X., Zhong, C., Huang, Y., Zhou, R., Shi, S. (2015) De novo assembly of the transcriptomes of two yellow mangroves, *Ceriops tagal* and *C. zippeliana*, and one of their terrestrial relatives, *Pellacalyx yunnanensis*. ***Marine Genomics***, 23: 33-36. PMID: 25899405. ([link](https://www.sciencedirect.com/science/article/abs/pii/S1874778715000586))

**Yang, Y.**, Yang, S., Fang, L., Li, J., Zhong, C., Zhou, R., Shi, S. (2015) Phylogenetic position of *Sonneratia griffithii* based on sequences of the nuclear ribosomal internal transcribed spacer and 13 nuclear genes. ***Journal of Systematics and Evolution***, 53(1): 47-52. ([link](https://onlinelibrary.wiley.com/doi/abs/10.1111/jse.12102))

